### PR TITLE
DELETE /v1/contextEntities/EID now uses sdt-op and not conv-map

### PR DESCRIPTION
**Pull request by kzangeli**
One more convop moved to use std-op service routine.

### CODE REVIEW

File: src/lib/serviceRoutines/deleteIndividualContextEntity.cpp [hunk 1]
```diff
@@ -49,17 +70,30 @@ std::string deleteIndividualContextEntity
...
-  std::string  entityId = "unknown entityId";
+  std::string  entityId   = compV[2];
+  std::string  entityType = ciP->uriParam[URI_PARAM_ENTITY_TYPE];
   StatusCode   response;
 
-  if (compV.size() > 2)
+  // 01. Fill in UpdateContextRequest fromURL-data + URI params
+  parseDataP->upcr.res.fill(entityId, entityType, "false", "DELETE");
+  
+  LM_M(("KZ: Fill in UpdateContextRequest id:'%s', type='%s'", entityId.c_str(), entityType.c_str()));
```
Comment (fgalan) [inline src/lib/serviceRoutines/deleteIndividualContextEntity.cpp]: Leftover?

File: src/lib/serviceRoutines/deleteIndividualContextEntity.cpp [hunk 2]
```diff
@@ -49,17 +70,30 @@ std::string deleteIndividualContextEntity
...
+
+  // 02. Call postUpdateContext standard service routine
+  answer = postUpdateContext(ciP, components, compV, parseDataP);
+  LM_M(("KZ: answer: '%s'", answer.c_str()));
```
Comment (fgalan) [inline src/lib/serviceRoutines/deleteIndividualContextEntity.cpp]: Leftover?

Comment (kzangeli) [inline src/lib/serviceRoutines/deleteIndividualContextEntity.cpp]: Of course, I forgot to run my grep LM_M ...

Comment (kzangeli) [inline src/lib/serviceRoutines/deleteIndividualContextEntity.cpp]: Fixed in 7ae539e

File: src/lib/serviceRoutines/deleteIndividualContextEntity.cpp (hunk 1 above)
Comment (kzangeli) [inline src/lib/serviceRoutines/deleteIndividualContextEntity.cpp]: Fixed in 7ae539e

**Comment by fgalan** [comment]
LGTM

**Pull request merged**